### PR TITLE
[ko] Update outdated files in dev-1.21-ko.1 branch (p1)

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Related issue: #27490

> 81. [ ]  M81.  `content/en/docs/tasks/job/parallel-processing-expansion.md`  | 1(+XS) 1(-)
> 82. [ ]  M82.  `content/en/docs/tasks/manage-daemon/rollback-daemon-set.md`  | 1(+XS) 1(-)
> 83. [ ]  M83.  `content/en/docs/tasks/manage-daemon/update-daemon-set.md`  | 2(+XS) 2(-)
> 84. [ ]  M84.  `content/en/docs/tasks/manage-gpus/scheduling-gpus.md`  | 1(+XS) 1(-)
> 85. [ ]  M85.  `content/en/docs/tasks/run-application/delete-stateful-set.md`  | 2(+XS) 2(-)
> 86. [ ]  M86.  `content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md`  | 1(+XS) 1(-)
> 87. [ ]  M87.  `content/en/docs/tasks/run-application/horizontal-pod-autoscale.md`  | 3(+XS) 5(-)
> 88. [ ]  M88.  `content/en/docs/tasks/tools/_index.md`  | 3(+XS) 3(-)
> 89. [ ]  M89.  `content/en/docs/tutorials/_index.md`  | 2(+XS) 0(-)
> 90. [ ]  M90.  `content/en/docs/tutorials/clusters/apparmor.md`  | 2(+XS) 2(-)
> 91. [ ]  M91.  `content/en/docs/tutorials/hello-minikube.md`  | 4(+XS) 4(-)
> 92. [ ]  M92.  `content/en/docs/tutorials/services/source-ip.md`  | 1(+XS) 1(-)
> 93. [ ]  M93.  `content/en/docs/tutorials/stateful-application/basic-stateful-set.md`  | 1(+XS) 1(-)
> 94. [ ]  M94.  `content/en/docs/tutorials/stateless-application/guestbook.md`  | 25(+S) 24(-)
> 95. [ ]  M95.  `content/en/examples/application/job/cronjob.yaml`  | 1(+XS) 1(-)
> 96. [ ]  M96.  `content/en/examples/application/zookeeper/zookeeper.yaml`  | 1(+XS) 1(-)

### CODE REVIEW

**Comment by seokho-son** [comment]
@ClaudiaJKang 리뷰 가능하실까요~? ^^
/assign @ClaudiaJKang

File: content/ko/docs/tutorials/stateless-application/guestbook.md [hunk 1]
```diff
@@ -49,13 +49,15 @@ min-kubernetes-server-version: v1.14
 1. 매니페스트 파일을 다운로드한 디렉터리에서 터미널 창을 시작한다.
 1. `mongo-deployment.yaml` 파일을 통해 MongoDB 디플로이먼트에 적용한다.
 
+      <!--
+      컨텐츠에 대한 로컬 테스트는 파일의 상대경로로 한다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/stateless-application/guestbook.md:53]:
```suggestion
      콘텐츠에 대한 로컬 테스트는 파일의 상대 경로로 한다.
```

- 외래어 표기법 상 '콘텐츠'로 쓰고 있습니다.

File: content/ko/docs/tutorials/stateless-application/guestbook.md [hunk 2]
```diff
@@ -84,14 +86,15 @@ kubectl apply -f ./content/en/examples/application/guestbook/mongo-deployment.ya
 
 1. `mongo-service.yaml` 파일을 통해 MongoDB 서비스에 적용한다.
 
+      <!--
+      컨텐츠에 대한 로컬 테스트는 파일의 상대경로로 한다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/stateless-application/guestbook.md:90]:
```suggestion
      콘텐츠에 대한 로컬 테스트는 파일의 상대 경로로 한다.
```

File: content/ko/docs/tutorials/stateless-application/guestbook.md [hunk 3]
```diff
@@ -122,14 +125,15 @@ kubectl apply -f ./content/en/examples/application/guestbook/mongo-service.yaml
 
 1. `frontend-deployment.yaml` 파일을 통해 프론트엔드의 디플로이먼트에 적용한다.
 
+      <!--
+      컨텐츠에 대한 로컬 테스트는 파일의 상대경로로 한다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/stateless-application/guestbook.md:129]:
```suggestion
      콘텐츠에 대한 로컬 테스트는 파일의 상대 경로로 한다.
```

**Review by pjhwa — COMMENTED** [review]
@jihoon-seo 지나다가 몇 가지 의견 드립니다. ^^

**Comment by jihoon-seo** [comment]
@pjhwa 리뷰 감사합니다.
알려 주신 사항들을 수정하고 squash, force-push 하였습니다.
감사합니다.. ^^

**Comment by seokho-son** [comment]
@ClaudiaJKang 님은 이번에 리뷰가 어려우신 것 같습니다.
/unassign @ClaudiaJKang 

@pjhwa 님께서 지나가신 김에.. Assign 해도 될까요? ^0^
/assign @pjhwa

**Comment by pjhwa** [comment]
그럼 제가 /lgtm 해도 될까요? ^^

**Comment by seokho-son** [comment]
@gochist 님~~ 해당 PR의 확인 및 승인 부탁드립니다.. ^^

**Pull request merged**